pkg/domain/entities/containers.go

```diff
@@ -178,6 +178,7 @@ type CheckpointOptions struct {
 	TCPEstablished bool
 	PreCheckPoint  bool
 	WithPrevious   bool
+	Compression    int64
```

Comment:
```suggestion
	Compression    archive.Compression
```
Why not use the compression type vs. converting to and from int64?

Reply:
Initially I was using `archive.Compression`, but then I thought that it seems a bit much to include another package just to track 3 integer values. As I am also using this in my attempt to bring checkpoint/restore to CRI-O and I am including `libpod.CheckpointOptions` there and so `int64` seemed simpler. But `archive` is already part of Podman and CRI-O so that it doesn't really make a difference. It just felt like a too large dependency for 3 integer values. But I can easily change it back to `archive.Compression`.

Reply:
In for a penny, in for a pound, you are already including the package so I would prefer the code being type safe. Thanks.

Reply:
Switched everything to `archive.Compression`.